controller/workitem.go

```diff
@@ -316,7 +317,13 @@ func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 		Fields: make(map[string]interface{}),
 	}
 	return application.Transactional(c.db, func(appl application.Application) error {
-		err := ConvertJSONAPIToWorkItem(ctx, appl, *ctx.Payload.Data, &wi, ctx.SpaceID)
+		//verify spaceID:
+		// To be removed once we have endpoint like - /api/space/{spaceID}/workitems
+		_, spaceLoadErr := appl.Spaces().Load(ctx, ctx.SpaceID)
```

Comment:
If you merge today this #1415 PR then you should replace this Load by CheckExists() function